examples/src/ThumbnailCardExamples.js

```diff
@@ -48,7 +57,11 @@ const ThumbnailCardExamples = () => {
                     {new Array(3).fill(true).map((_, i) => (
                         <ThumbnailCard
                             key={`thumbnailcard-${i}`}
+                            className={isCardInFocus === i ? 'is-card-in-focus' : ''}
                             icon={icon}
+                            id={i}
+                            setIsCardInFocus={setIsCardInFocus}
+                            shadowOnHover
```

Comment:
we should use a prop name that describes the state it is in, and not how that state is currently implemented. Think `highlightOnHover` or `showFocusState` or similar.

Reply:
I think that makes sense. Let's go with `highlightOnHover`.